src/universal/config/myarea-datasets.ts

```diff
@@ -1,8 +1,8 @@
 import { FeatureToggle } from './app';
 
 // Amsterdam center
-export const DEFAULT_LAT: number = 52.3717228;
-export const DEFAULT_LNG: number = 4.8927377;
+export const DEFAULT_LAT: number = 52.3676842478192;
```

Comment:
Als ik lokaal de BRP delay en deze coördinaten naar een andere plek in Amsterdam zet dan wordt de kaart geladen op die coördinaten en niet Amstel 1 (waar onze mock user woont). 

Volgens mij ligt de oorzaak van het probleem er in dat `@amsterdam/react-maps` niet reageert op geupdate locatie props. En wij pakken in `MyAreaDashboard` deze default zolang `My_Location` nog aan het laden is.

Reply:
Hmm. ga ik naar kijken